docs/Privacy/Start-Pantheon-Privacy.md

```diff
@@ -1,111 +1,95 @@
 ```
 
 ```bash tab="Example Configuration File"
-privacy-precompiled-address=125
+nodeurl = "http://127.0.0.1:8080/"
+nodeport = 8080
+nodenetworkinterface = "127.0.0.1"
+clienturl = "http://127.0.0.1:8888/"
+clientport = 8888
+clientnetworkinterface = "127.0.0.1"
+workdir = "data"
+othernodes = ["http://127.0.0.1:8081/"]
+publickeys = ["key.pub"]
+privatekeys = ["key.key"]
 ```
 
-Address to which the privacy pre-compiled contract is mapped.
-The default is 126. 
-
 !!!note
-    Privacy is under development and will be available in v1.1.    
-
+    You must have an Orion Enclave running with the `nodeurl` set to `http://127.0.0.1:8081/`
+    and `nodeport` set to `8081` for the above configuration to work.
+    
+!!!note
+    You do not need to specify `othernodes` for the first Orion instance that is created.
 
-### Enclave (Orion)
+### Starting Pantheon
 
-#### privacy-public-key-file
+Below is an example of the minimum configuration to set-up a Pantheon node 
+capable of sending and processing Private Transactions.
 
 ```bash tab="Syntax"
---privacy-public-key-file=<privacyPublicKeyFile>
+pantheon --network=<NETWORK> --rpc-http-api=<api name>[,<api name>...]... --privacy-enabled[=<true|false>] --privacy-url=<url> --privacy-public-key-file=<FILE>
 ```
 
-Path to the public key for the enclave.     
-
-!!!note
-    Privacy is under development and will be available in v1.1.
-
-!!!note
-    This option is not used when running Pantheon from the [Docker image](../Getting-Started/Run-Docker-Image.md#privacy-public-key-file).
-
-#### privacy-url
+```bash tab="Example Command Line"
+pantheon --network=dev --rpc-http-api=EEA --privacy-enabled --privacy-url=http://meenclave.com/8888 --privacy-public-key-file=/home/me/me_enclave/data/key
+```
 
-```bash tab="Syntax"
---privacy-url=<privacyUrl>
+```bash tab="Example Configuration File"
+network="dev" 
+rpc-http-api="EEA" 
+privacy-enabled=true 
+privacy-url="http://meenclave.com/8888"
+privacy-public-key-file="/home/me/me_enclave/data/key"
 ```
 
-URL on which the Enclave is running.    
+More information about the option used above can be found via the following links:
+- [network](../Reference/Pantheon-CLI-Syntax.md#network)
+- [rpc-http-api](../Reference/Pantheon-CLI-Syntax.md#rpc-http-api)
+- [privacy-enabled](../Reference/Pantheon-CLI-Syntax.md#privacy-enabled)
+- [privacy-url](../Reference/Pantheon-CLI-Syntax.md#privacy-url)
+- [privacy-public-key-file](../Reference/Pantheon-CLI-Syntax.md#privacy-public-key-file)
 
 !!!note
     Privacy is under development and will be available in v1.1.
     
-    
-### Privacy JSON-RPC API method
-
-The [EEA methods](../Reference/JSON-RPC-API-Methods.md#eea_sendRawTransaction) were created to
-provide and support privacy.
-
-### Set-up Privacy
-
-## Prerequisites 
-
-[Pantheon](../Installation/Install-Binaries.md) 
-
-[Curl (or similar web service client)](https://curl.haxx.se/download.html) 
-
-## Steps
+## Private Transaction walkthrough
```

Comment:
I have reservations to write a walkthrough in the main Pantheon docs. I prefer to create a separate project (pantheon-privacy-examples or something) that will have the basic structure set-up. Then we can just link that project here and put docs about it.

In addition - I would like to question the relevance of having this walkthrough in the first-place. Quorum does not have such a walkthrough and I do not think it is popular enough to run multiple nodes on one local machine. I would like to point out here that we are also working on a dockerized environment for showcasing this. I feel I should put my attention on documentation there.

Does anyone strongly feel that the information below this heading is useful?

Reply:
The information below the heading is more about describing the configuration (command line options, Orion config file etc) needed to set up multiple nodes rather than running multiple nodes on one machine (the nodes could be on different machines and the configuration steps are still required).  I'm going to do some updates to it to focus it on the configuration that's specific to setting up privacy which should cut down the overall size. I'm happy to own updating/rework etc on the privacy documentation merged so far. And agree it makes sense for you to focus on the documentation around the dockerized environment.

Reply:
So to restate what Madeline just said, having seperate documentation that helps people get their own environment setup is useful, and the point of the documentation.  I think that this type of documentation makes sense, and that Madeline would do a great job of writing it.

Reply:
Perfect. So I have already outlined the basic command line arguments you need to set-up a Pantheon and Orion node on a local machine. Good luck with the rest! :)